fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4246,29 +4248,47 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     checkEFOBBelowMin() {
+        if(this._fuelPredDone) {
         if (!this._minDestFobEntered) {
             this.tryUpdateMinDestFob();
         }
-        const EFOBBelMin = this.isAnEngineOn() ? this.getDestEFOB(true) : this.getDestEFOB(false);
 
-        if (EFOBBelMin < this._minDestFob) {
-            if (this.isAnEngineOn()) {
-                setTimeout(() => {
-                    this.addMessageToQueue(NXSystemMessages.destEfobBelowMin, () => {
-                        return this._EfobBelowMinClr === true;
-                    }, () => {
-                        this._EfobBelowMinClr = true;
-                    });
-                }, 180000);
+        if(this._minDestFob) {
+            // round & only use 100kgs precision since thats how it is displayed in fuel pred
+            const EFOBBelMin = Math.round( (this.isAnEngineOn() ? this.getDestEFOB(true) : this.getDestEFOB(false)) * 10) / 10;
```

Comment:
Formatting. Also, this is really just EFOB at destination, isn't it?
```suggestion
            const destEfob = Math.round(this.getDestEFOB(this.isAnEngineOn()) * 10) / 10;
```

Reply:
Yup